### PR TITLE
[TASK] Remove unused docker images with ansible

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

File: ansible/webapp.yml [hunk 1]
```diff
@@ -39,6 +39,9 @@
       copy: src=private/dhparam.pem dest={{nginx_dh_params}}
       notify:
         - reload nginx container
+  post_tasks:
+    - name: Remove unused docker images
+      command: docker rmi `docker images -q`
```
Comment (stevenzeiler) [inline ansible/webapp.yml]: Will this fail the circle tests if there is an error?

**Comment by stevenzeiler** [comment]
:+1:

File: ansible/webapp.yml (hunk 1 above)
Comment (wilsonianb) [inline ansible/webapp.yml]: I've just added `ignore_errors` for when docker fails try remove images currently used by running containers

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**